builder/Dockerfile.alpine-tracee-make

```diff
@@ -0,0 +1,31 @@
+FROM alpine:3.15 as alpine-tracee-make
+
+ARG uid=1000
+ARG gid=1000
+
+RUN apk --no-cache update && \
+    apk --no-cache add sudo coreutils findutils && \
+    apk --no-cache add bash git curl && \
+    apk --no-cache add musl-dev libc6-compat && \
+    apk --no-cache add llvm clang go make gcc && \
+    apk --no-cache add linux-headers && \
+    apk --no-cache add elfutils-dev && \
+    apk --no-cache add libelf-static && \
+    apk --no-cache add zlib-static && \
+    curl -L -o /usr/bin/opa https://github.com/open-policy-agent/opa/releases/download/v0.33.1/opa_linux_amd64_static && \
+    chmod 755 /usr/bin/opa
+
+RUN export uid=$uid gid=$gid && \
```

Comment:
Yes, I don't know if the UID/GID picked by the builder user will exist in the distro picked to be our builder. By adding those lines to the end of passwd & group file I'm sure they will be the ones picked (instead of having to deal with the logic of checking if those UID/GID already exist).